src/test/java/org/jenkinsci/plugins/vsphere/tools/CloudProvisioningStateTest.java

```diff
@@ -233,7 +233,7 @@ public void pruneUnwantedRecordsGivenUnknownTemplatesThenRemovesRecordsForEmptyD
         final String deletedButActiveNodeName = createNodeName();
         final String livedAndDiedNodeName = createNodeName();
         final CloudProvisioningState instance = createInstance();
-        // A template which the user deleted but still has an active slave
+        // A template which the user deleted but still has an active agent 
```

Comment:
Please remove trailing whitespace

Reply:
Remove trailing whitespace from the src/test/java/org/jenkinsci/plugins/vsphere/tools/CloudProvisioningStateTest.java file.